app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] As per the project requirements, the application is supposed to ask for confirmation of email. For this functionality to work, `:confirmable` is supposed to be enabled which is not the case right now, Please kindly enable :`confirmable.`
```
devise :database_authenticatable, :registerable,
				:recoverable, :rememberable, :validatable, :confirmable
```